helpers/calendar/analytics_helper.py

```diff
@@ -172,16 +173,35 @@ def get_room_statistics(self, number_of_events_in_room, all_details):
         return result
 
     @staticmethod
-    def get_total_bookings(self, query, start_date, end_date):
+    def get_total_bookings(instance, query, start_date, end_date, id=None):
```

Comment:
Kindly explain why you changed this from `self` to `instance`

Reply:
@RachaelNantale static methods should not have `self` as the first positional argument hence having it there would cause confusion. To avoid this, I renamed it to`instance` since it's just a variable and would give more context on what's going on.